src/utils/packageResult.js

```diff
@@ -12,17 +12,26 @@ export default function packageResult ( body, options, methodName, isBundle ) {
 	code = body.toString();
 
 	if ( !!options.sourceMap ) {
-		if ( !options.sourceMapFile || ( !isBundle && !options.sourceMapSource )  ) {
-			throw new Error( 'You must provide `sourceMapSource` and `sourceMapFile` options' );
+		if ( options.sourceMap !== 'inline' && !options.sourceMapFile) {
+			throw new Error( 'You must provide `sourceMapFile` option' );
 		}
 
-		let sourceMapFile = isAbsolutePath( options.sourceMapFile ) ? options.sourceMapFile : './' + splitPath( options.sourceMapFile ).pop();
+		if ( !isBundle && !options.sourceMapSource ) {
+			throw new Error( 'You must provide `sourceMapSource` option' );
+		}
+
+		let sourceMapFile;
+		if (options.sourceMap === 'inline') {
+			sourceMapFile = null;
```

Comment:
@rwjblue Having this set to `null` throws an error when attempting to set `sourceMap:'inline',` through `esperontoOptions` in `EmberApp.prototype.javascript` since there is no "source" value in the `content.sources` array when assimilation occurs in `fast-sourcemap-concat`.

Setting the value to `input.js.map` allows the sourcemap to generate properly from esperonto.

Is there a better way of getting ES6 source out of ember-clim or one in which overriding `EmberApp.prototype.javascript` is not required where this is not an issue?